src/components.rs

```diff
@@ -46,6 +46,60 @@ pub struct Arsc {
     pub global_string_pool: StringPool,
 }
 
```

Comment:
I do not really understand what this PR is for. The functions are helper functions that can be easily done on the user end. There does not seem to be a reason to add them to the library